driver/terraform.go

```diff
@@ -550,10 +494,10 @@ func (tf *Terraform) initTaskTemplate() error {
 //
 // Returned handler may be nil even if returned err is nil. This happens when
 // no providers have a handler.
-func getTerraformHandlers(task Task) (handler.Handler, error) {
+func getTerraformHandlers(name string, providers TerraformProviderBlocks) (handler.Handler, error) {
```

Comment:
I like that you changed this to only pass in what's needed! Optional thought - someone using this function might not know what `name` refers to without looking into the function's details (e.g. it might be handler name) - would something like taskName work ok?

Reply:
done! thanks again